enterprise/backend/src/metabase_enterprise/metabot_v3/tools/change-chart-appearance.json

```diff
@@ -0,0 +1,119 @@
+{
```

Comment:
same file with `current_display_type` changed to `display_type` because this is how we send it now